synapseclient/evaluation.py

```diff
@@ -34,28 +32,26 @@
     - **OPEN** indicating processing *has not* completed
     - **CLOSED** indicating processing *has* completed
 
-Submission status objects can be updated, usually by changing the *status* and *score* fields, and stored back to
-Synapse using :py:func:`synapseclient.Synapse.store`::
+Submission status objects can be updated, usually by changing the `status` and `score` fields, and stored back to
+Synapse using [`synapseclient.Synapse.store`][synapseclient.Synapse.store]:
 
     status.score = 0.99
     status.status = 'SCORED'
     status = syn.store(status)
 
 See:
 
-- :py:func:`synapseclient.Synapse.getEvaluation`
-- :py:func:`synapseclient.Synapse.getEvaluationByContentSource`
-- :py:func:`synapseclient.Synapse.getEvaluationByName`
-- :py:func:`synapseclient.Synapse.submit`
-- :py:func:`synapseclient.Synapse.getSubmissions`
-- :py:func:`synapseclient.Synapse.getSubmission`
-- :py:func:`synapseclient.Synapse.getSubmissionStatus`
-- :py:func:`synapseclient.Synapse.getPermissions`
-- :py:func:`synapseclient.Synapse.setPermissions`
-
-~~~~~~~~~~
-Evaluation
-~~~~~~~~~~
+- `synapseclient.Synapse.getEvaluation`
```

Comment:
These should be links using cross-reference: https://mkdocstrings.github.io/usage/#cross-references

To auto link to the applicable function in the python docs. This is similar to what you've got in the file:
```suggestion
- [synapseclient.Synapse.getEvaluation][]
```

Test this out. But I think this was the proper format you can use if you don't want to display different text.


You can use this same format above as well. I don't thing we need to show all the function text in a `code block`